### PR TITLE
Release v0.0.0-20200508 

### PR DESCRIPTION
**Pull request by DC7806**
@RocketSH 請抽空將目前在 develop 所有 PR 內做事項條列在這

- Project Initialize

  - rails initialize #2
  - update README.md #6 
<br />

- Setup CSS framework and packages

  - install Bootstrap, jQuery, axios/ turn off turbolinks/ Procfile for foreman #7
  - correct bootstrap version #15 
<br />

- Setup test framework
  -  Add rspec
  - install shoulda-matchers #8
<br />

- Add database.yml into .gitignore

  - add database.yml.example #11

### CODE REVIEW

**Comment by koten0224** [comment]
找不到database.yml.sample

**Comment by koten0224** [comment]
沒有database.yml.sample

**Comment by koten0224** [comment]
等等，database.yml.example還是database.yml.sample ?

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+web: bin/rails server -p 3000
+webpack: bin/webpack-dev-server
```
Comment (ErenWeng) [inline Procfile:2]: 這是什

File: Procfile 2 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+web: bin/rails server -p 3000
+webpack: bin/webpack-dev-server
```
Comment (ErenWeng) [inline Procfile 2:2]: 重複出現

**Pull request merged**